Fix: prevent ENOENT crash in mvnw.js by checking file existence

amirbekazimov opened the pull request:
## Fix: prevent ENOENT crash in mvnw.js when counting files

### Problem
Running `eoc compile` sometimes crashed with the following error:
Error: ENOENT: no such file or directory, stat '.eoc/classes/EOorg/EOeolang/EOnan$EOtimes.class'
at Object.statSync (node:fs:1658:25)
at count (/src/mvnw.js:170:16)


This happened because `mvnw.js` was calling `fs.statSync` on files that might have already been removed or not yet generated, which caused the **ENOENT** error.

### Solution
- Added a check with `fs.existsSync` before calling `fs.statSync`.
- Wrapped file counting in safe error handling so the process no longer crashes if a file is missing.
- Now the status printer (`print()`) works reliably during compilation.

### Verification
- Created a sample project with `hello.eo`.
- Ran:
  ```bash
  node ./src/eoc.js compile -s ../test-co

yegor256 commented:
@amirbekazimov many thanks for your contribution! Please, fix eslint warnings and we'll merge. Also keep in mind that we don't allow empty lines inside method bodies and don't allow comments inside methods.

amirbekazimov commented:
> @amirbekazimov many thanks for your contribution! Please, fix eslint warnings and we'll merge. Also keep in mind that we don't allow empty lines inside method bodies and don't allow comments inside methods.

Thanks for the feedback! I've fixed all ESLint warnings, removed empty lines and comments inside methods as requested. The PR is now ready for review.

yegor256 commented:
@amirbekazimov this is a nice code polishing, but I don't think it will fix the bug. Thanks for the contribution anyway!

yegor256 commented:
@amirbekazimov keep in mind that in order to fix a bug we need to reproduce it in a test first

0crat commented:
@amirbekazimov **Great work on your contribution!** 🎉 You've earned **+12 points** (+16 base, -4 for 32 hits-of-code). [Keep the momentum going](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) - your running score is now [+12](https://www.eolang.org/zerocracy/objectionary-vitals.html). Don't forget to check your [Zerocracy account](https://www.zerocracy.com/) for updates!

<!-- Thanks for the contribution! You've earned +12 points for this: +16 as a basis; -4 for too few (32) hits-of-code. Please, [keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) them coming. Your running score is [+12](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).  -->
